linux_os/guide/services/ssh/ssh_server/sshd_rekey_limit/oval/shared.xml

```diff
@@ -0,0 +1,43 @@
+{{% set filepath = "/etc/ssh/sshd_config" %}}
+{{% set parameter = "RekeyLimit" %}}
+
+
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    <metadata>
+      <title>{{{ rule_title }}}</title>
+      {{{- oval_affected(products) }}}
+      <description>Ensure '{{{ RekeyLimit }}}' is configured with the correct value in '{{{ filepath }}}'</description>
```

Comment:
Jinja variable `parameter` is not used anywhere.

> `'{{{ RekeyLimit }}}'`

I guess it should be `{{{ parameter }}}`, or explicitly `RekeyLimit`.

Reply:
Done, it was an artifact of the earlier check evolution.